tilelang/autodd.py

```
@@ -0,0 +1,1170 @@
+
+    def post_proc(self, text):
+        return clean_empty_pass(text)
+
+    async def worker(self, wid: int):
+        runner = AsyncPythonRunner() if self.backend == "runner" else SubProcRunner()
+        try:
+            while True:
+                task = await self.get_next_task()
+                if task is None:
+                    break
+                out, err, ok = await runner.run(task.source, timeout=self.timeout)
```

Ruff (0.14.10)

Unpacked variable `ok` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)